Makefile

```diff
@@ -12,6 +12,7 @@ all:
 	make test-report
 	make build
 	make sonar
+
```

Comment:
why introduce a blank line when other targets do not have one?

Reply:
yeah i ll remove vijay